db/migrate/20160908062011_process_images_for_listing_cards.rb

```diff
@@ -0,0 +1,25 @@
+class ProcessImagesForListingCards < ActiveRecord::Migration
+
+  def up
+    select_values("SELECT id FROM listing_images").each_slice(1000) { |ids|
+
+      exec_insert("INSERT INTO delayed_jobs
+                  (priority, handler, last_error, run_at, created_at, updated_at, queue)
+                  VALUES #{values(ids)}", "create_delayed_jobs", [])
```

Comment:
Is there a risk that the massive insert locks the table for too long time? That would have immediate effect on the application performance.